sdk/formrecognizer/azure-ai-formrecognizer/src/test/resources/session-records/DocumentModelAdminClientTest.beginBuildClassifierWithJsonL[1].json

```diff
@@ -0,0 +1,193 @@
+      "Date" : "Tue, 28 Mar 2023 00:01:23 GMT",
+      "Content-Type" : "application/json; charset=utf-8"
+    },
+    "Exception" : null
+  }, {
+    "Method" : "GET",
+    "Uri" : "https://REDACTED.cognitiveservices.azure.com/formrecognizer/operations/31486406334_dc20f2b3-6cb9-4ee6-bd0c-e900ccf457ad?api-version=2023-02-28-preview",
+    "Headers" : {
+      "User-Agent" : "azsdk-java-azure-ai-formrecognizer/4.1.0-beta.1 (11.0.9; Windows 10; 10.0)",
+      "x-ms-client-request-id" : "92bc83f3-0a27-453e-9435-e33f25c0d5ac"
+    },
+    "Response" : {
+      "Transfer-Encoding" : "chunked",
+      "x-ms-region" : "East US",
+      "x-envoy-upstream-service-time" : "24",
+      "Strict-Transport-Security" : "max-age=31536000; includeSubDomains; preload",
+      "x-content-type-options" : "nosniff",
+      "apim-request-id" : "2dbef234-dbd6-4864-8ea1-b213ebd246e6",
+      "retry-after" : "0",
+      "StatusCode" : "200",
+      "Body" : "{\n  \"operationId\": \"31486406334_dc20f2b3-6cb9-4ee6-bd0c-e900ccf457ad\",\n  \"kind\": \"documentClassifierBuild\",\n  \"status\": \"running\",\n  \"createdDateTime\": \"2023-03-28T00:01:06Z\",\n  \"lastUpdatedDateTime\": \"2023-03-28T00:01:17Z\",\n  \"resourceLocation\": \"https://shafangfr.cognitiveservices.azure.com/formrecognizer/documentClassifiers/ae8c0d77-933a-4b1b-83d8-d824923f2706?api-version=2023-02-28-preview\"\n}",
+      "Date" : "Tue, 28 Mar 2023 00:01:28 GMT",
+      "Content-Type" : "application/json; charset=utf-8"
+    },
+    "Exception" : null
+  }, {
+    "Method" : "GET",
+    "Uri" : "https://REDACTED.cognitiveservices.azure.com/formrecognizer/operations/31486406334_dc20f2b3-6cb9-4ee6-bd0c-e900ccf457ad?api-version=2023-02-28-preview",
+    "Headers" : {
+      "User-Agent" : "azsdk-java-azure-ai-formrecognizer/4.1.0-beta.1 (11.0.9; Windows 10; 10.0)",
+      "x-ms-client-request-id" : "45bcac87-7332-4d55-b8f2-62a579d66452"
+    },
+    "Response" : {
+      "Transfer-Encoding" : "chunked",
+      "x-ms-region" : "East US",
+      "x-envoy-upstream-service-time" : "21",
+      "Strict-Transport-Security" : "max-age=31536000; includeSubDomains; preload",
+      "x-content-type-options" : "nosniff",
+      "apim-request-id" : "13f6d5b1-7682-4773-a92f-9d4cb04595f9",
+      "retry-after" : "0",
+      "StatusCode" : "200",
+      "Body" : "{\n  \"operationId\": \"31486406334_dc20f2b3-6cb9-4ee6-bd0c-e900ccf457ad\",\n  \"kind\": \"documentClassifierBuild\",\n  \"status\": \"running\",\n  \"createdDateTime\": \"2023-03-28T00:01:06Z\",\n  \"lastUpdatedDateTime\": \"2023-03-28T00:01:17Z\",\n  \"resourceLocation\": \"https://shafangfr.cognitiveservices.azure.com/formrecognizer/documentClassifiers/ae8c0d77-933a-4b1b-83d8-d824923f2706?api-version=2023-02-28-preview\"\n}",
```

Comment:
Nothing related to this PR. but for the TestRedaction keyword, "resourceLocation" is also one that should be considered.

Reply:
yeah good point, I will keep it for custom sanitizations though maybe.